boa/src/vm/opcode.rs

```diff
@@ -0,0 +1,614 @@
+use std::convert::TryFrom;
+
+/// The opcodes of the vm.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[repr(u8)]
+pub enum Opcode {
+	/// Pop the top value from the stack.
+	///
+	/// Operands:
+	///
+	/// Stack: value **=>**
+    Pop = 0,
```

Comment:
Why is `Pop` `0` but the rest doesn't have a number assigned?

Reply:
https://doc.rust-lang.org/reference/items/enumerations.html#custom-discriminant-values-for-fieldless-enumerations Basically it will force Rust to set the remaining variants to auto-increment the number from that point onwards.

However, their docs do say that if a number isn’t specified it will apply 0 by default, so the `= 0` may be unnecessary unless it’s supposed to be indicative

Reply:
As @jasonwilliams said it's not necessary, just wanted to make sure it started at `0`. I'll remove it :)